oqs-sys/build.rs

```diff
@@ -117,7 +114,39 @@ fn main() {
     }
     println!("cargo:rustc-link-search=native={}", libdir.display());
 
-    let gen_bindings = |file, filter| generate_bindings(&outdir, file, filter);
+    outdir
+}
+
+fn includedir_from_source() -> PathBuf {
+    let outdir = build_from_source();
+    outdir.join("build").join("include")
+}
+
+fn probe_includedir() -> PathBuf {
+    if cfg!(feature = "vendored") {
+        return includedir_from_source();
+    }
+
+    let lower_bound = env!("CARGO_PKG_VERSION");
```

Comment:
This might be any value in `0.8.100..0.8.199`, to refer to the versions of this wrapper that packages lib's `0.8.1` per our versioning scheme in the README. 

So we need this lower bound to be restricted a bit less by taking only the first character of the patch version.

Reply:
I don't know how possible it is that liboqs reaches version e.g. 0.8.10, but I'd probably apply logic like:

if patch version is at least 3 chars long, cut out the last two

Reply:
that sounds reasonable 👍🏻